http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeConfigTable.scala

```diff
@@ -45,17 +47,23 @@ class RuntimeConfigTable(tag: Tag) extends Table[RuntimeConfigRecord](tag, "RUNT
              workerDiskSize,
              numberOfWorkerLocalSSDs,
              numberOfPreemptibleWorkers,
-             dataprocProperties),
+             dataprocProperties,
+             persistentDiskId),
             dateAccessed) =>
         val r = cloudService match {
           case CloudService.GCE =>
-            RuntimeConfig.GceConfig(machineType, diskSize)
-
+            diskSize match {
+              case Some(size) => RuntimeConfig.GceConfig(machineType, size)
+              case None =>
+                persistentDiskId.fold(
+                  RuntimeConfig.GceWithPdConfig(machineType, None)
```

Comment:
and should this be just `GceConfig`?

Reply:
no...see comment above

Reply:
but there is no `persistentDiskId` in this case right?

Reply:
yea..that just means the disk is detached (when runtime is deleted)